app/assets/javascripts/controllers/angular-bootstrap/DatepickerCtrl.js

```diff
@@ -0,0 +1,22 @@
+miqAngularApplication.controller('DatepickerCtrl', function ($scope) {
+  $scope.minDate = new Date();
+
+  $scope.disabled = function(date, viewDate, mode) {
+    disabledDate = new Date($scope.$parent.angularForm[viewDate].$viewValue);
+
+    if($scope.$parent.angularForm[viewDate].$untouched) {
+      return ( mode === 'day' && (disabledDate.getUTCDate() === date.getDate()) &&
+                              disabledDate.getUTCMonth() === date.getMonth() &&
+                              disabledDate.getUTCFullYear() === date.getFullYear())
```

Comment:
As someone whose local date is right now != UTC date, this is relevant to my interests :smile:

As @himdel said, this Just Sounds Weird. If it is in fact correct, I think it'd be worth a comment explaining where things are getting out of whack, and why this is the right thing to do after all.